### DISCUSSION TITLE
Release of `0.2.0`

### DISCUSSION THREAD

**Discussion by ted-miller** [Yaskawa-Global/motoros2#120] [General]
In context of https://github.com/Yaskawa-Global/motoros2/pull/49#issuecomment-1677769935.

For a 0.2.0 release, I'd suggest we also try to get the [latest humble](https://github.com/Yaskawa-Global/micro_ros_motoplus/pull/117) and [iron support](https://github.com/Yaskawa-Global/micro_ros_motoplus/issues/115#issue-1772812356) working.

The focus of `0.2.0` would be increasing compatibility.

**Comment by gavanderhoorn** [comment]
So would you want to release `0.1.1` with "just" DX200 compatibility, or release it with DX200 compatibility *and* the updates you mention (and the updates would be `0.2.0`)?

**Reply by ted-miller** [reply]
I was suggesting we don't do an official release until iron has been integrated (and humble updated). At that time we would release 0.2.0.

But as I'm typing this, I'm realizing that not everyone has M+SDK to build the DX200 version from source. So, it would actually be beneficial to make a DX200 binary release sooner. 

So, I guess we should just finalize the 0.1.1 milestone and make a release.

**Reply by gavanderhoorn** [reply]
Ok. Let's prepare `0.1.1` to release the MR2 binaries with DX200 support.

I don't see a need to bump minor just for Iron support tbh then btw.

**Comment by gavanderhoorn** [comment]
@ted-miller: give the amount of time that has passed and the difficulties Iron compatibility appears to pose, I would like to suggest we *repurpose* `0.2.0` and use it to only update the used M+ `libmicroros` versions (for Humble) and use it to release all the improvements made since `0.1.2` ([diff](https://github.com/Yaskawa-Global/motoros2/compare/0.1.2...main)).

I can contribute the updated M+ `libmicroros`.

**Reply by ted-miller** [reply]
That's fine with me. Are you wanting to do that before or after #265?

And do you really want to repurpose `0.2.0`, or just release an intermediary `0.1.3`? (I don't care either way. But the code is already marked 013pre.)

**Reply by gavanderhoorn** [reply]
Good point. We can just release `0.1.3` instead.
